src/components/staking/PageValidator.vue

```diff
@@ -200,7 +204,7 @@ import moment from "moment"
 import { calculateTokens } from "scripts/common"
 import { mapGetters } from "vuex"
 import num, { atoms, viewDenom, shortDecimals } from "scripts/num"
-import { formatBech32 } from "src/filters"
+import { formatBech32, expectedReturns } from "src/filters"
```

Comment:
I mostly intended for `src/filters` to be used for actual Vue filters. Simple formatting etc. Maybe there's a better folder structure.